ansible/scripts/install_ansible.sh

```diff
@@ -22,9 +22,19 @@ base_dir=$( cd "$( dirname "$bin" )" && pwd )
 
 set -e
 
+# get the release id
+if [ -f /etc/os-release ]; then
+  . /etc/os-release
+  ID=$ID
+fi
+
 # enable yum epel repo
-is_installed_epel_release="rpm -q --quiet epel-release"
-install_epel_release="sudo yum install -q -y epel-release"
+if [ $ID = "rhel" ]; then
+  is_installed_epel_release="rpm -q --quiet epel-release"
```

Comment:
Seems like `is_installed_epel_release` is only set when ID is rhel.  Is that ok?